relational_operators/TextScanOperator.cpp

```diff
@@ -288,7 +288,7 @@ void TextScanWorkOrder::execute() {
 
     std::fclose(file);
   } else {
-    BlobReference blob = storage_manager_->getBlob(text_blob_);
+    MutableBlobReference blob = storage_manager_->getBlobMutable(text_blob_);
```

Comment:
Because the `release` method below modifies `blob` by doing some cleanups, i.e., destroying the `CountedReference`.